gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DocumentReferenceMapper.java

```diff
@@ -68,7 +68,7 @@ private DocumentReference mapDocumentReference(RCMRMT030101UK04NarrativeStatemen
         documentReference.getMeta().addProfile(META_PROFILE);
         documentReference.setStatus(DocumentReferenceStatus.CURRENT);
         documentReference.setType(getType(narrativeStatement));
-        documentReference.setSubject(new Reference(patient.getId()));
+        documentReference.setSubject(new Reference(patient));
```

Comment:
Is this just something you noticed that needed fixed?

Reply:
Yes :) In the result, the reference String didn't contain object name like "Patient/some_id_here" but only the id